compose-builder/README.md

```diff
@@ -55,8 +55,24 @@ This folder contains the following compose files:
     Device Service **extending** compose file, which adds the **Device SNMP**  service.
 - **add-device-virtual.yml**
     Device Service **extending** compose file, which adds the **Device Virtual**  service.
+- **add-asc-http-export.yml**
+    Application Service Configurable **extending** compose file, which adds the **App Service Http Export**  service.
+- **add-asc-http-export-secure.yml**
+    Application Service Configurable **extending** compose file, which adds the **App Service Http Export Secrets** service. Service is enable for secure secrets.
+- **add-asc-mqtt-export.yml**
+    Application Service Configurable **extending** compose file, which adds the **App Service MQTT Export**  service.
+- **add-asc-MQTT-export-secure.yml**
+    Application Service Configurable **extending** compose file, which adds the **App Service Http Export Secrets** service. Service is enable for secure secrets.
+- **add-modbus-simulator.yml**
+    ModBus Simulator **extending** compose file. Adds the MQTT ModBus Simulator service. Must be used in conjunction with  **add-device-modbus.yml**
+- **add-mqtt-broker.yml**
+    MQTT Broker **extending** compose file. Adds the Eclipse Mosquitto MQTT Broker.
 - **add-mqtt-messagebus.yml**
-    MQTT **extending** compose file. Adds a MQTT Broker and additional configuration of services so that the `MQTT` implementation of the Edgex Message Bus is used.
+    MQTT MesssageBus **extending** compose file. Adds additional configuration of services so that the `MQTT` implementation of the Edgex Message Bus is used. Must be use in conjunction with **add-mqtt-broker.yml**
```

Comment:
```suggestion
    MQTT MesssageBus **extending** compose file. Adds additional configuration of services so that the `MQTT` implementation of the Edgex Message Bus is used. **Must be used in conjunction with add-mqtt-broker.yml**
```
would prefer to stress on the whole must be part